eventuals/generator.h

```diff
@@ -305,13 +307,13 @@ struct _Generator final {
     K_ k_;
   };
 
-  template <typename From_, typename To_, typename Errors_, typename... Args_>
+  template <typename From_, typename To_, typename Catches_, typename Errors_, typename... Args_>
   struct Composable final {
-    template <typename Arg>
+    template <typename Arg, typename Errors>
     using ValueFrom = To_;
 
     template <typename Arg, typename Errors>
-    using ErrorsFrom = tuple_types_union_t<Errors, Errors_>;
+    using ErrorsFrom = tuple_types_union_t<Errors_, tuple_types_unique_t<Errors, Catches_>>;
```

Comment:
We want to subtract `Catches_` from `Errors` and then union that with `Errors_`!

Reply:
`tuple_types_subtract`